README.md

```diff
@@ -21,19 +21,54 @@ see https://stackoverflow.com/questions/71262775/how-do-i-ensure-assets-are-pres
   - postgres
   - yarn
 
-You can install postgres with `brew install postgresql` if you have homebrew. Then please run `bundle exec rails db:create` to set up the development and test databases.
+## Setting up the app
 
-Install dependencies:
+You can use [Homebrew](https://brew.sh/) to install any dependencies. The Homebrew documentation has lots of useful commands.
+
+If using Homebrew:
+
+Install postgres. You will need to select a version if using brew. Run the below command, changing the version number as appropriate.
+
+```
+brew install postgresql@14
+```
+
+You will be prompted on the command line to start the server with something like:
 
 ```
+brew services start postgresql@14
+```
+
+Install [Bundler](https://bundler.io/). You may run into an error where you need to update your Ruby version. Use rbenv or another version manager to do this.
```

Comment:
I _believe_ that as of Ruby 2.6, Bundler is built into Ruby, so you don't actually have to do a `gem install bundler` to get bundler to be available for a `bundle install`. That being said, flagging up that you need to have the correct version of Ruby installed - specifically 3.1.3 at the moment. It might be worth updating line 17 to make that explicit.